docs/adr/0004-principles.md

```diff
@@ -0,0 +1,30 @@
+# Team Principles
+
+Date: 15.06.18
+
+## Status
+
+Proposed
+
+## Context
+
+We need a set of common default principles to govern how the team will use GitHub to organise its own workflow.
+
+## Decision
+
+* We will prioritise reviewing Pull Requests (PRs) over beginning new issues.
+* Where possible, issues will include a user story and product requirements/acceptance criteria as well as links to relevant designs.
+* Where possible, issues will be grouped together according to the product feature they relate to.
+* After two days' effort, an assignee will add a comment to an issue and/or consider raising new issues to clearly indicate progress.
+* When feedback is required, an assignee will open a PR with the 'WIP' prefix. See [this ADR](https://github.com/elifesciences/elife-xpub/blob/develop/docs/adr/0001-pull-request-workflow.md).
+* As we facilitate deployment on a per-branch basis, once a PR is merged it is considered 'Done'.
+* Further end-to-end testing will be raised in a new issue with the 'e2e test' label.
```

Comment:
If a feature can naturally be tested as part of an existing end-to-end test flow, then I think it should usually be done in the same issue.